Mathlib/Data/Rat/Cast/Lemmas.lean

```diff
@@ -0,0 +1,41 @@
+/-
+Copyright (c) 2019 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro
+-/
+import Mathlib.Data.Rat.Cast.Defs
+import Mathlib.Algebra.Field.Basic
+
+#align_import data.rat.cast from "leanprover-community/mathlib"@"acebd8d49928f6ed8920e502a6c90674e75bd441"
+
+
+/-!
+# Some exiled lemmas about casting
+
+These lemmas have been removed from `Mathlib.Data.Rat.Cast.Defs`
+to avoiding needing to import `Mathlib.Algebra.Field.Basic` there.
+
+In fact, these lemmas don't appear to be used anywhere in Mathlib,
+so perhaps this file can simply be deleted.
```

Comment:
I'd be happy with deleting this file, but I'll leave it to your call.

Reply:
I'd prefer to leave this for someone who knows these lemmas better to delete later.